.github/workflows/linux-build.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           sudo apt-get update
           sudo apt-get install libusb-1.0-0 libusb-1.0-0-dev
       - run: ./bootstrap
-      - run: ./configure --enable-remote-bitbang --enable-jtag_vpi --enable-ftdi-oscan1
+      - run: ./configure --enable-remote-bitbang --enable-jtag_vpi --enable-ftdi-cjtag
```

Comment:
@timsifive - Do you please know what the tradition is in upstream for the switches of `./configure`?

Can we rename them directly? Or should the old one get deprecated but left as an alias for some time?

Reply:
I don't recall any discussion like that. I did check and internally at SiFive it'll be OK. Personally I think this change is a good one. If upstream wants to keep the old version supported, then that's probably a small change we can make when it's time to upstream.

Reply:
If upstream is this: https://github.com/openocd-org/openocd, OScan1 was never added to it